multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/OperationsApiServiceImpl.java

```diff
@@ -250,6 +251,8 @@ private Operation addServiceParameters(Operation operation, String spaceGuid, St
         parameters.put(Variables.ORGANIZATION_GUID.getName(), organization.getMetadata()
                                                                           .getGuid()
                                                                           .toString());
+        parameters.put(Variables.TIMESTAMP.getName(), DateTimeFormatter.ofPattern("yyyyMMddHHmmss")
```

Comment:
Why this format?

Reply:
It's the same format as before, taken from CollectSystemParametersStep